wireapp/outlook-addin
# fix(API access): Update API from v2 to v5 and add API Version as env

----
#### PR Submission Checklist for internal contributors

- The **PR Title**

- The **PR Description**
----

# What's new in this PR?

### Issues

The currently used Wire-API version is v2. Needed to be changed to v5.
- Linear Ticket Url: [https://linear.app/wire-integrations/issue/WIR-12/switch-from-wire-api-v2-to-wire-api-v5](https://linear.app/wire-integrations/issue/WIR-12/switch-from-wire-api-v2-to-wire-api-v5)

### Solutions

Added new environment variable to quickly change the API version in future issues. Used this variable to specify which API version to access.

### Testing

#### How to Test

Check the network tab in the dev-tools to verify the selected API version.

----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.
